diff --git a/src/scripts/install_asdf.bash b/src/scripts/install_asdf.bash
index 7a4b680..d3d1ad8 100644
--- a/src/scripts/install_asdf.bash
+++ b/src/scripts/install_asdf.bash
@@ -8,10 +8,7 @@
 ###
 
 SCRIPT_PATH="$([ -L "$0" ] && readlink "$0" || echo "$0")"
-SCRIPT_DIR="$(
-  cd "$(dirname "${SCRIPT_PATH}")" || exit 1
-  pwd -P
-)"
+SCRIPT_DIR="$(cd "$(dirname "${SCRIPT_PATH}")" || exit 1; pwd -P)"
 ROOT_DIR="$(cd "${SCRIPT_DIR}/../.." && pwd)"
 SHELLPACK_DEPS_DIR="${SHELLPACK_DEPS_DIR:-"${ROOT_DIR}/.shellpack_deps"}"
 SHELL_GR_DIR="${SHELL_GR_DIR:-"${SHELLPACK_DEPS_DIR}/@github/rynkowsg/shell-gr@02965b2cbbe4707c052f26eb90aac9308816c94b"}"
